Source/WebCore/accessibility/AccessibilityScrollView.cpp

```diff
@@ -214,6 +214,9 @@ void AccessibilityScrollView::clearChildren()
 
 bool AccessibilityScrollView::computeIsIgnored() const
 {
+    if (m_remoteFrame)
```

Comment:
Could we please add a comment explaining why having a remote frame means we shouldn't ignore this scroll view?